deepchem/models/torch_models/layers.py

```diff
@@ -0,0 +1,514 @@
+import math
```

Comment:
This file shouldn't be in the current PR

Reply:
Oh looks like that got added in by mistake. I'll close this PR and open up a fresh one.